src/gevent_server.py

```diff
@@ -0,0 +1,32 @@
+# -*- coding: utf-8 -*-
+from gevent.server import StreamServer
+from gevent.monkey import patch_all
```

Comment:
I think it'd be better to isolate these imports from gevent in the `main` block below.  That way you could import this server function without needing to import the gevent tools at all.

However, this is not a show-stopper.